src/prommis/uky/costing/ree_plant_capcost.py

```diff
@@ -3660,11 +3660,10 @@ def verify_calculate_from_inputs(b):
 
             for key in costs.keys():
                 # check if the object is a Reference
-                try:
-                    costs[key] = costs[key][None]
-                except:
-                    # continue on
-                    pass
+                if type(costs[key]) not in [float, int]:
```

Comment:
Checking types like this is rather brittle. For example, this method would fail if `costs[key]` was a numpy float of some sort. The `isinstance` function checks whether a variable is a member of a class or any of its subclasses and is thus more robust for things like this. I'd `from pyomo.core.base.component import Component` and then check `isinstance(costs[key], Component)` to see if the variable is a Pyomo modeling component.